release-engineering/role-handbooks/branch-manager.md

```diff
@@ -327,51 +329,43 @@ If the CI results are good and the release team approves, you can initiate the p
 
 This presumes reproducible builds and that the CI tests are meaningful relative to the release builds. There is always a risk that these diverge, and this needs to be managed broadly by the project and the release team.
 
-##### krel gcbmgr release
+##### krel release
 
 Before running the release step, please refer to the [Image Promotion documentation](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/release-image-promotion.md).
 
-Use the `--build-version=` as specified in the output when `krel gcbmgr` is done with the stage command.
-
-To run `krel gcbmgr` to release an Alpha release see [here](https://github.com/kubernetes/release/blob/master/docs/krel/gcbmgr.md#alpha-release).
+To finish the previously staged release, run `krel release --build-version=…`.
+Use the `--build-version=` as specified in the output when `krel stage` is done.
```

Comment:
> Use the `--build-version=` as specified in the output when `krel stage` is done.

Nit: we should make it more clear what's the output, e.g. we can say something like `in the GCB logs` or `at the end of the GCB logs`.

Reply:
Hm, yeah I can follow-up on that :+1: